sources/packages/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -110,4 +117,21 @@ export class ScholasticStandingInstitutionsController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Get Scholastic Standing submitted details.
+   * @param scholasticStandingId scholastic standing id.
+   * @returns Scholastic Standing.
+   */
+  @Get(":scholasticStandingId")
+  @ApiNotFoundResponse({
+    description: "Scholastic Standing not found.",
+  })
+  async getScholasticStanding(
```

Comment:
This is an institutions-api. Should it not be validated for institutionId?

Reply:
There is no need to use `location/:locationId` to enforce the authorization.
We an also keep the endpoint URI as `api/institutions/scholastic-standing/:scholasticStandingId` and do the authorizations as below:
![image](https://user-images.githubusercontent.com/61259237/174897404-d9669773-8824-43e6-8899-dfcebf289e7c.png)

Reply:
The change looks good to me. Since was @dheepak-aot that added the comment I will leave for him the final evaluation.